src/main/java/uk/ac/ox/oucs/sirlouie/LibraryAvailability.java

```diff
@@ -49,9 +49,9 @@ public Response get(@QueryParam("id") String id, @QueryParam("format") @DefaultV
 
 		try {
 
-			PrimoService service = new PrimoService(getProperties().getWebResourseURL());
-			DaiaURI uri = new DaiaURI(id);
-			ResponseBean bean = service.getResource(uri.getDoc());
+			String webResourceURL = getProperties().getWebResourseURL(id);
+			PrimoService service = new PrimoService(webResourceURL);
```

Comment:
Shouldn't this be called `AlephService` instead as it's no longer designed for talking to Primo?